pkg/scheduling/volumeusage.go

```diff
@@ -132,7 +122,7 @@ func resolveDriver(ctx context.Context, kubeClient client.Client, volumeName str
 	}
```

Comment:
Should we fail to sync nodes if we fail to find the PV that is referenced by the PVC? Is it possible for a PVC to reference a PV that doesn't exist and the pod still to be bound?

Reply:
Yes, if the PV is dynamically provisioned via a storage class the PV is not provisioned until the pod is bound.